index.html

```diff
@@ -0,0 +1,12 @@
+<html>
+    <head>
+        <h4>Tooltips</h4>
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+    <body>
+        <div tooltip="tooltip text" tooltip-position="top">Top</div>
```

Comment:
Text on tooltip shouldn't be static, it should change according to the text passed within tooltip-text attribute e.g. `<div tooltip="Something else..." tooltip-position="top">Top</div>`

Reply:
not getting